api/Pipfile

```diff
@@ -35,11 +37,10 @@ elasticsearch-dsl = "~=8.9"
 future = "~=0.18"
 limit = "~=0.2"
 Pillow = "~=10.1.0"
+psycopg = "~=3.1"
 python-decouple = "~=3.8"
 python-xmp-toolkit = "~=2.0"
 sentry-sdk = "~=1.30"
-django-split-settings = "*"
-psycopg = "~=3.1"
```

Comment:
Just moving things into alphabetical order here. The only meaningful change is to add adrf at the top.